### DISCUSSION TITLE
ios打开网站，视频播放器ui界面溢出

### DISCUSSION THREAD

**Discussion by Dashan-IZ** [AlistGo/alist#4033] [Q&A]
ios打开网站，视频播放器ui界面溢出，这个怎么解决，3.15版本
![微信图片_20230403214149](https://user-images.githubusercontent.com/78125220/229529039-8f6c5848-de32-423e-a2a9-9b4cbf37e084.jpg)

网站：specialcloud.top
ios用户帮忙试试看看是不是有问题

**Comment by Dashan-IZ** [comment]
https://specialcloud.top/

**Comment by anwen-anyi** [comment]
https://github.com/alist-org/alist/issues/3954 别急下个版本就好了
